src/BootstrapTable/assets/src/controller.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+import 'bootstrap/dist/css/bootstrap.min.css'
+import 'bootstrap';
```

Comment:
I don't think the end-user wants to import the whole Bootstrap CSS/JS code to render a table.

Reply:
This was also the first part of the PR that jumped out at me. A few things:

1) My guess is that we don't need to import bootstrap from inside here. If `bootstrap-table` uses `bootstrap`, IT will import it internally.

2) We also shouldn't import `bootstrap.min.css`. That'll be up to the user to make sure they're importing this, or importing the parts they want :).

3) We probably don't want to rely on the "data attributes" way of initializing the functionality. We should initialize on `this.element` directly.

Reply:
I agree with both of you on the import of the css and js. I put that here more for the demo. But I am not sure about your third point @weaverryan. Do you mean instead of using data attributes we should generate everything by javascript? And if yes why?

Reply:
Yes! Exactly! I don't think that relying on the `data-` attributes will actually work if that HTML is loaded later via Ajax onto the page. That's why, with all the other UX components, we leverage Stimulus for this. So, on the table, you would output something like `data-controller="symfony--ux-bootstrap-table--controller"` to connect to this controller.  You would also pass in the data via a "value" (just like chart.js does). Then, in this controller, in the `connect()` method, you would initialize BootstrapTable programmatically - e.g.

```js
$(this.element).bootstrapTable(this.tableDataValue).
```

Reply:
Ok fine! I commit that this week-end